pkgs/tools/X11/inputplug/default.nix

```diff
@@ -0,0 +1,32 @@
+{ fetchCrate
+, installShellFiles
+, lib
+, libbsd
+, pkg-config
+, rustPlatform
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "inputplug";
+  version = "0.4.0";
+
+  src = fetchCrate {
+    inherit pname version;
+    sha256 = "00gv2i2pxy56l6ysslbscxinr4r0mpk9p2ivkrnjnwhc8j3v8v7h";
+  };
+
+  nativeBuildInputs = [ installShellFiles pkg-config ];
+  buildInputs = [ libbsd ];
+
+  cargoHash = "sha256:161kz47d4psfvh0vm98k8qappg50lpsw1ybyy7s3g3bp6ivfz8jv";
```

Comment:
change to `cargoSha256`

Reply:
```suggestion
  cargoHash256 = "161kz47d4psfvh0vm98k8qappg50lpsw1ybyy7s3g3bp6ivfz8jv";
```

Reply:
it should be `cargoSha256` not `cargoHash256`